docs/en_us/platform_api/source/conf.py

```diff
@@ -186,10 +186,10 @@
     'xmodule.course_module',
     'user_api.accounts.api',
     'user_api.accounts.serializers',
-    'ecommerce_api_client',
+    'edx_rest_api_client',
```

Comment:
In case others are as confused as I was about what's going on here, see [this entry](http://read-the-docs.readthedocs.org/en/latest/faq.html#i-get-import-errors-on-libraries-that-depend-on-c-modules) from ReadTheDocs' FAQ.

ImportErrors are raised when Sphinx's autodoc module tries to import modules that aren't present in the standard library, and documentation can't be compiled if the code being documented isn't importable. A cleaners way to solve this is with a requirements file for ReadTheDocs, but this also works.